canonical-web-and-design/maas-ui
# feat(ui): mark completed_intro in the user intro

## Done

- Update the user intro to set completed_intro. 
- Add actions and reducers and status and error handlers for updating completed_intro.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Create a new user.
- Log in as the new user and go to /MAAS/r/intro/user
- Click "Finish setup".
- You should get redirected once it has updated.
- Visit /MAAS/r/intro/user again and you should get redirected immediately.

## Fixes

Fixes: canonical-web-and-design/app-squad#162.
